src/main/java/com/notably/logic/commands/AddCommand.java

```diff
@@ -63,4 +64,4 @@ public boolean equals(Object other) {
                 || (other instanceof AddCommand // instanceof handles nulls
                 && toAdd.equals(((AddCommand) other).toAdd));
     }
-}
+}
```

Comment:
You need a newline character after the end of the file. Otherwise, this will account to a style error (although we can't really check right now since the build is failing)